### DISCUSSION TITLE
[공통 컴포넌트] ProfileImage

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#7] [General]
<img width="123" alt="스크린샷 2022-06-07 오후 4 39 08" src="https://user-images.githubusercontent.com/72402747/172323524-032c00e0-7d7f-415d-aeeb-82473758d3a7.png">

**Comment by YuHyun-P** [comment]
## props
### imgSrc
### imgAlt
### size
### onClick
